pkg/agent/cniserver/server.go

```diff
@@ -433,6 +433,58 @@ func (s *CNIServer) CmdAdd(ctx context.Context, request *cnipb.CniCmdRequest) (*
 			return s.ipamFailureResponse(err), nil
 		}
 	}
+	// TODO(gran): remove this hack
```

Comment:
do you think we could remove the below in this PR, or should it be done in a follow-up PR?